executor/show_stats.go

```diff
@@ -186,6 +192,36 @@ func (e *ShowExec) bucketsToRows(dbName, tblName, partitionName, colName string,
 	return nil
 }
 
+func (e *ShowExec) indexBucketsToRows(dbName, tblName, partitionName, colName string, numOfCols int, hist statistics.Histogram, idxColumnType []byte) error {
```

Comment:
How about adding the new argument `idxColumnType` to the original function `bucketsToRows` instead of creating a new one?
And the argument `idxColumnType` can be set to `nil`.